### DISCUSSION TITLE
Night Photo or Night Time-lapse Photo Preset FAILURE

### DISCUSSION THREAD

**Discussion by MatthewPoland** [gopro/labs#244] [General]
Hi, I'm using GoPro labs for a while (more or less 2 years) and still running into same issue with Night Photo modes.
It's no matter if I'm using Photo Mode set to Night Photo or Time lapse Mode set to Night time-lapse, very time the camera (GoPro 8 or 9) is about to take some photos during the "day" it hangs! 
No qrlog event either.
Is it because of long exposure metering, buffering or what?
My thought is: do not use the Night modes! if you want to use Labs for time-lapse or time delay functionality - because it will fail.
Stick to standard photo mode or time-lapse mode and you are good to go!

Question: Is there an option to make a time-lapse from night (long exposure) to day (high shutter speeds)?
What I came up is to try to use ND filters to prevent from higher shutter speeds, but it will affect the night photos as well, 
e.g. causing longer exposures than intervals. 
Are there any other solutions?

**Comment by dnewman-gpsw** [comment]
Never seen that.  What firmware are you running?  Please share the qr code that causes the crash.  I use Night mode all the time, day or night.

**Reply by Videocamguy** [reply]
What interval are you using for the TimeLapse?

**Reply by MatthewPoland** [reply]
I think I'm running on firmware v01.70.00 with latest labs firmware.
The example code: 
No1 `mNPoW1!S!18:00E!U!07:30R` - if power consumption is irrelevant, interval 30s.
No2 `mPN>7:29<17:59!SQ~!7:30S!60R` - for energy efficient, interval 60s.
In each example the camera hangs, sometimes in half of the day, sometimes even after 60 photos.
When I switch to Photo Timelapse or Photo mode everything is fine.
I'm using !MBOOT command to load scripts, also !MWAKE for power trigger and reset GoPro after hangs.
I tried power cycle the GoPro after 18:00 to start time-lapse (No1 code) immediately and it run through the night flawlessly and hangs when the daytime starts, never last to reach !U or !18:00E

**Reply by dnewman-gpsw** [reply]
If 1.70.00 is neither a Labs firmware version or a current version on any camera.  The version number is the camera menus under Prefs->About and on the SD Card under MISC/version.txt.

HERO9 is at v1.72.70 
HERO8 is at v2.51.75

Commands of style No2, will hang randomly on any camera.  I'm not recommend that style any model.

No1 should work fine. This is no reason it would stop in the middle of the day (other than hardware failures camera or SD.) As you can test by enabled Night Lapse Photo and pressing capture.  The script has one oddity, turning the WiFi at the beginning may not work as it might time-out before the capture end. mNP!S!18:00EoW1!1U!07:30R  Might be better.
